lib/database/models/user.js

```diff
@@ -32,6 +32,7 @@ module.exports = (sequelize) => {
 
     User.associate = (models) => {
         User.hasMany(models.Log, { foreignKey: 'userId', as: 'logs' });
+        User.hasMany(models.QualityControlFlag, { as: 'qualityControlFlags', foreignKey: 'userId' });
```

Comment:
Do you have a use for this? If not, I would rather not put it.

Reply:
ok, right, named association here is not needed

Reply:
users and flags are associated so, here is definition of association

Reply:
If I am not mistaken, `hasMany` is not mandatory, we only need the owning side. Here, we don't use this side for now, I would rather drop it

Reply:
applied